omod/src/main/webapp/fragments/patientHeader.gsp

```diff
@@ -113,7 +113,13 @@
                     <% } else { %>
                         ${ui.message("coreapps.ageDays", patient.ageInDays)}
                     <% } %>
-                    (<% if (patient.birthdateEstimated) { %>~<% } %>${ ui.formatDatePretty(patient.birthdate) })
+                    (<% if (patient.birthdateEstimated) { %>~<% } %>
```

Comment:
thanks @icrc-jfigueiredo  cc  @dkayiwa

Reply:
@icrc-jfigueiredo did you test these changes by running the module and see the user interface effects?

Reply:
Yes, I did test it, with Timzone GP as `true` and `false`, and it works well.

Reply:
@icrc-jfigueiredo did you test it before or after removing the extra `formatDatePretty` that you had below?

Reply:
After removing it.

Reply:
But I think we should release a new version of UiFramework, because we are using a method that is only available on the snapshot version.